packages/ui/src/components/Button/index.tsx

```diff
@@ -257,19 +255,9 @@ const SmartIcon = ({
     <Icon name={icon as ComponentProps<typeof Icon>['name']} size={iconSize} />
   )
 
-type StyledIcon = {
-  margin: number
-  position?: 'left' | 'right'
-}
-
-const StyledIconContainer = styled('div', {
-  shouldForwardProp: prop => !['margin', 'position'].includes(prop),
-})<StyledIcon>`
+const StyledIconContainer = styled('div')`
```

Comment:
```suggestion
const StyledIconContainer = styled.div`
```

Not much difference it is just shorter